lib/chef/resource/aws_cloudsearch_domain.rb

```diff
@@ -0,0 +1,46 @@
+require 'chef/provisioning/aws_driver/aws_resource'
+
+module AWS
+  class CloudSearch
+    class Domain
+      # The version of the AWS sdk we are using doesn't have a model
+      # object for CloudSearch Domains. This empty class is here to
+      # make the reset of chef-provisioning happy.
+    end
+  end
+end
+
+class Chef::Resource::AwsCloudsearchDomain < Chef::Provisioning::AWSDriver::AWSResource
+  aws_sdk_type ::AWS::CloudSearch::Domain
+  attribute :domain_name, kind_of: String, name_attribute: true
+  attribute :cloudsearch_api_version, equal_to: ["20130101", "20110201"], default: "20130101"
```

Comment:
Ahh - I see why you did this.  Their [docs](http://docs.aws.amazon.com/AWSRubySDK/latest/AWS/CloudSearch/Client.html) specify that 2011 is the default because 2013 is not backwards compatiable.